src/main/kotlin/apply/ui/api/AdministratorRestController.kt

```diff
@@ -0,0 +1,50 @@
+package apply.ui.api
+
+import apply.application.AdministratorData
+import apply.application.AdministratorResponse
+import apply.application.AdministratorService
+import apply.domain.user.User
+import apply.security.LoginUser
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import support.toUri
+import javax.validation.Valid
+
+@RequestMapping("/api/admin/administrators")
```

Comment:
c: 'administrators' 앞에 'admin'이 필요한가요?

```suggestion
@RequestMapping("/api/administrators")
```